README.md

```diff
@@ -54,6 +54,109 @@ $ go get -u github.com/skroutz/mistry/cmd/mistry
 
 
 
+Examples
```

Comment:
We could move this whole section under a README in the `examples` directory and just add a link to that readme in here:

> For example projects take a look at the [examples](examples/) directory.

Reply:
done